backend/small_eod/generic/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+# Create your models here.
+from django.conf import settings
+
+
+class TimestampUserLogModel(models.Model):
+
+    createdBy = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.DO_NOTHING,
+        related_name='%(class)s_createdBy',
+        null=True,
+        blank=True
+    )
+    modifiedBy = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.DO_NOTHING,
+        related_name='%(class)s_modifiedBy',
+        null=True,
+        blank=True
+    )
+
+    modifiedOn = models.DateTimeField(auto_now=True)
```

Comment:
Proponuje użyć ```TimestampedModel``` z ```djanog-model-utils```, bo m. in. w mojej ocenie pole powinno mieć ```editable=False```.

Reply:
auto_now i auto_now_add automatycznie dodaja editable=False. TimestampedModel nie obsluguje z tego co widze createdBy i modifiedBy (tylko createdOn i modifiedOn) - wiec musialbym z niego dziedziczyc i utworzyc kolejna klase posrednia, zeby utworzyc te pola zwiazane z Userem. Stworzylem wiec swoja, niezalezna od django-model-utils (generic.models.TimestampUserLogModel). 
Przeanalizuj to prosze i daj znac gdyby musialo byc zmienione.